podium-gateway/src/main/java/nl/thehyve/podium/service/mapper/RequestDetailMapper.java

```diff
@@ -0,0 +1,18 @@
+package nl.thehyve.podium.service.mapper;
+
+import nl.thehyve.podium.domain.RequestDetail;
+import nl.thehyve.podium.service.representation.RequestDetailRepresentation;
+import org.mapstruct.Mapper;
+
+/**
```

Comment:
This is not the correct license. Use Apache v2 license header already present in the project.